### ISSUE TITLE
C857 question

### ISSUE THREAD

**Issue by rudolphh** [jmatzen/quizsail#27]
In a walkthrough the majority of program testing is conducted by the author of the program.

Isn't this true?

**Comment by jmatzen** [comment]
I don't have access to the source material anymore, but I remember this question specifically.  Does the book say something differently?

**Comment by drewwmercer** [comment]
From Lesson 3.1:
> In a walkthrough, a group of developers—with three or four being an optimal number—performs the review. Only one of the participants is the author of the program. Therefore, **the majority of program testing is conducted by people other than the author**, which follows testing principle 2, which states that an individual is usually ineffective in testing his or her own program.

**Comment by rudolphh** [comment]
>  I don't have access to the source material anymore, but I remember this question specifically.  Does the book say something differently?

You're right. I was just using the web which says the author does it, but the books says since other developers take part then the "majority" of the testing done is not by the author. Thanks.

**Comment by SSimp258** [comment]
According to the CMMI reading in the course material (Kasse, T. (2008). Practical Insight into CMMI. (2nd ed.) Artech House Publishers. for reference).
![Untitled](https://user-images.githubusercontent.com/96361739/157372826-a6ba41ed-84a9-4465-a94b-a38a07e68547.png)
.

**Comment by SSimp258** [comment]
I should also point out I have come across multiple questions that are wrong similar to this one.

**Comment by jmatzen** [comment]
> I should also point out I have come across multiple questions that are wrong similar to this one.

It would be cool if you submitted a fix for them.

**Comment by SSimp258** [comment]
i'm not familiar with github and i have no idea how you guys have this setup. I've had to email my course instructors on the questions that are wrong or don't make sense but they haven't gotten back to me. The question could be correct because there is the other book in the course material. I just remember where this is located in this book and went right to it.
